cmd/podman/inspect.go

```diff
@@ -287,7 +288,7 @@ type HostConfig struct {
 	ContainerIDFile      string                      `json:"ContainerIDFile"`
 	LogConfig            *LogConfig                  `json:"LogConfig"` //TODO
 	NetworkMode          string                      `json:"NetworkMode"`
-	PortBindings         map[string]struct{}         `json:"PortBindings"` //TODO
+	PortBindings         nat.PortMap                 `json:"PortBindings"` //TODO
```

Comment:
Almost certainly not going to end up being this, as we use different structs elsewhere. I would recommend `[]ocicni.PortMapping` as used in libpod

Reply:
Ok, I am just trying to use the parsing code from Docker which returns this field.